lib/ansible/module_utils/mysql.py

```diff
@@ -28,13 +28,34 @@
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import os
+from ansible.module_utils.six import (
+    PY2
+)
 
 try:
     import MySQLdb
     mysqldb_found = True
 except ImportError:
     mysqldb_found = False
 
+if PY2:
+    try:
+        import configparser
+    except ImportError:
+        import ConfigParser as configparser
+else:
+    import configparser
```

Comment:
This can be directly imported from six:
``` python
from ansible.module_utils.six.moves import configparser
```

Reply:
@abadger I changed my code. Thanks for great review! :)